eng/Packages.Data.props

```diff
@@ -134,8 +134,8 @@
     <PackageReference Update="Microsoft.Azure.SignalR.Management" Version="1.19.2" />
     <PackageReference Update="Microsoft.Azure.SignalR.Protocols" Version="1.19.2" />
     <PackageReference Update="Microsoft.Azure.SignalR.Serverless.Protocols" Version="1.9.0" />
-    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.33" />
-    <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.30" />
+    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.36" />
+    <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.36" />
```

Comment:
@JoshLove-msft:  If I recall there was an issue in .36 and we had to stick with .35.   Can you confirm?

Reply:
@alrod should have context on this, I believe the issue was addressed last week?

Reply:
We were going to avoid bumping to 36 until we absolutely needed to. My understanding is that 36 has the scale controller support so we need to upgrade for this feature.